lib/std/coff.zig

```diff
@@ -256,20 +256,89 @@ pub const OptionalHeaderPE64 = extern struct {
     number_of_rva_and_sizes: u32,
 };
 
+pub const IMAGE_NUMBEROF_DIRECTORY_ENTRIES = 16;
+
+pub const DirectoryEntry = enum(u16) {
```

Comment:
Rationale behind `u16`:
* [ImageDirectoryEntryToData](https://docs.microsoft.com/en-us/windows/win32/api/dbghelp/nf-dbghelp-imagedirectoryentrytodata)'s parameter `DirectoryEntry` is `USHORT`
* [ImageDirectoryEntryToDataEx](https://docs.microsoft.com/en-us/windows/win32/api/dbghelp/nf-dbghelp-imagedirectoryentrytodataex)'s parameter `DirectoryEntry` is `USHORT`